tello Test/tello_web_server.py

```
@@ -651,15 +740,672 @@ def _log_operation(self, operation: str, details: Dict[str, Any]):
+                            self.ffmpeg_process.wait(timeout=2)
+                        except subprocess.TimeoutExpired:
+                            self.ffmpeg_process.kill()
+                        self.ffmpeg_process = None
+                        
+                except Exception as config_e:
+                    logger.warning(f"FFmpeg設定 {i+1} でエラー: {config_e}")
+                    if self.ffmpeg_process:
+                        try:
+                            self.ffmpeg_process.terminate()
+                            self.ffmpeg_process.wait(timeout=2)
+                        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)